utils/hostapi.d.ts

```diff
@@ -61,6 +61,13 @@ export interface AzureHostExtensionApi {
      */
     registerActivity(activity: Activity): Promise<void>;
 
+    /**
+     *
+     * @param filter Whether to filter the list returned, according to the list returned
+     * by `getTenantFilters()` and `getSubscriptionFilters()`. Optional, default true.
+     */
+    getSubscriptions: (filter: boolean) => Promise<AzureSubscription[]>;

```

Comment:
Should this be `(filter?: boolean)`?

Reply:
Oh yeah, I guess the comment does say that. I kind of wanted to be explicit about it so I'll amend the comment.